### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by isaka-lumato**
### FORMS

- Added functionality to add likes
- Added forms to input new Posts

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  #       format.html { render :show }
+  #     end
+  #   end
+  # end
+
+  # private
+
+  # def comment_params
+  #   params.require(:comment).permit(:text)
+  # end
```
Comment (elisha2kyakpo1) [inline app/controllers/comments_controller.rb]: - [x] Let's remove commented code to make the project look clean and readable

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  def new
```
Comment (elisha2kyakpo1) [inline app/controllers/comments_controller.rb:2]: - [x] I couldn't create a comment, after creating a post I didn't see a form for creating a comment. I kindly advise you fix this as well.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @isaka-lumato,

You have done a great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly update the readme file remove unwanted text, for example, the text on top was generated by Rails (when you created a new Rails app). Add more commands required for running the project like `bundle install`, `rails db:create` etc. Pay attention to grammar and spelling. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (isaka-lumato) [reply app/controllers/comments_controller.rb]: Thanks am gonna work on it asap

**Review by AshakaE — APPROVED** [review]
Hi @isaka-lumato , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**